package.json

```diff
@@ -10,6 +10,12 @@
       "esm"
     ],
     "exports": {
+      ".": {
+        "import": {
+          "types": "./dist/esm/session.d.ts",
+          "default": "./dist/esm/session.js"
```

Comment:
IMO it'd be less surprising if `session.ts` was `index.ts`

Reply:
done